imap_processing/glows/l0/decom_glows.py

```diff
@@ -64,11 +65,28 @@ def decom_packets(
                 apid = packet.header["PKT_APID"].derived_value
                 # Do something with the packet data
                 if apid == GlowsParams.HIST_APID.value:
-                    hist_l0 = HistogramL0(packet, version, filename)
+                    values = [
+                        item.derived_value
+                        if item.derived_value is not None
+                        else item.raw_value
```

Comment:
Personally I think a helper method for this is kind of overkill, especially since this is only in the decom steps. Presumably we could add something to space_packet_parser if we wanted, though